notify/teams.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env sh
+
+#Support Microsoft Teams webhooks
+
+#TEAMS_WEBHOOK_URL=""
+#TEAMS_THEME_COLOR=""
+#TEAMS_SUCCESS_COLOR=""
+#TEAMS_ERROR_COLOR=""
+#TEAMS_SKIP_COLOR=""
```

Comment:
Are all these  COLORs required? or optional?

Maybe we can give some default values to make it more easy-to-use.

For example, default color for Success can be green or yellow.
Error color can be RED
Skip color can be green or yellow.

Reply:
Colors are optional, I think we shouldn't set defaults.

Reply:
why not?
The user can only set one TEAMS_WEBHOOK_URL variable. it will work out-of-the-box.

Reply:
Done.